src/main/baseConnector.js

```diff
@@ -404,6 +449,14 @@ async function channelMessageHandler(message) {
                 dispatchEventLog(constants.MESSAGE_TYPE.GET_SIGNED_RECORDING_URL, signedRecordingUrlResult, true);
             }
         break;
+        case constants.MESSAGE_TYPE.DOWNLOAD_VENDOR_LOGS:
```

Comment:
Can we remove these two from here. These are only for Amazon Connector right now and once we figure out our 236 story for agent debugging, we can add them back